### DISCUSSION TITLE
Regrets of NodeJS by Ryan

### DISCUSSION THREAD

**Discussion by melroy89** [oven-sh/bun#5879] [Ideas]
The lessons learned from the NodeJS creator himself.. The regrets he has, could be a good idea to avoid this as much as possible @Jarred-Sumner ? So Bun will not repeat the same mistakes ;)

See the video by Ryan Dahl: https://youtu.be/M3BM9TB-8yA?si=dtgR_bDQB9rJqg9Y&t=305

**Comment by jimmywarting** [comment]
Kind of expressed my own feeling about the introduction of NodeJS stuff in bun also here myself: https://github.com/oven-sh/bun/discussions/5518
